Readme.mkd

```diff
@@ -81,6 +81,24 @@ Duck.rank(:row_order).first.row_order_rank # => 0
 Duck.rank(:row_order).third.row_order_rank # => 2
 ```
 
+The gem provides `{column_name}_rank_after!` method to move an item after a particular record in the list. `{column_name}` is the resource ranking column and the method accepts the previous item as a parameter.
```

Comment:
This gem provides the `{column_name}_rank_after!` method to position a record directly after another record in the list. `{column_name}` is the resource ranking column and the method accepts the previous record as a parameter.